### ISSUE TITLE
Halt simulation on assertion failure?

### ISSUE THREAD

**Issue by dgryski** [salesforce/runway-browser#8]
When running a model it's annoying to have to stop the simulation by hand and try to drag the state back to where the failure occurred.  A button that would halt the simulation immediately and allow exploring the current state would be easier. This also matches the behavior of the command line simulator.

**Comment by ongardie-sfdc** [comment]
Should we just halt all simulations upon an error? Then if the user clicks Simulate again, push forward?

**Comment by dgryski** [comment]
Perhaps buttons with "Run" and "Run until Error"?  Or a checkbox that says "Halt on error" (default to on? off?)

**Comment by ongardie-sfdc** [comment]
What's the use case for wanting to run past the error?

**Comment by dgryski** [comment]
No idea, was just trying to preserve somehow the current behaviour :)  If the current behaviour is unwanted, then yeah we don't need any of that.

**Comment by ongardie-sfdc** [comment]
Yeah, I don't think there's a reason for the current behavior. That's true for much of the UI :)
